junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvParserFactory.java

```diff
@@ -54,12 +54,13 @@ private static String selectDelimiter(Annotation annotation, char delimiter, Str
 	}
 
 	private static CsvParser createParser(String delimiter, String lineSeparator, char quote, String emptyValue,
-			int maxCharsPerColumn) {
-		return new CsvParser(createParserSettings(delimiter, lineSeparator, quote, emptyValue, maxCharsPerColumn));
+			int maxCharsPerColumn, boolean ignoreTrailingAndLeadingWhitespace) {
+		return new CsvParser(createParserSettings(delimiter, lineSeparator, quote, emptyValue, maxCharsPerColumn,
+			ignoreTrailingAndLeadingWhitespace));
 	}
 
 	private static CsvParserSettings createParserSettings(String delimiter, String lineSeparator, char quote,
-			String emptyValue, int maxCharsPerColumn) {
+			String emptyValue, int maxCharsPerColumn, boolean ignoreTrailingAndLeadingWhitespace) {
```

Comment:
Looks like you forgot to pass `ignoreTrailingAndLeadingWhitespace` to `CsvParserSettings`.

Reply:
Needed guidance there, do you want ignoreTrailingAndLeadingWhitespace as a property of CsvParserSettings. or does it translate to both of these booleans - setIgnoreTrailingWhitespaces() and setIgnoreLeadingWhitespaces()?

Reply:
Yes, the idea was to set both booleans based on the single flag from the annotation.

Reply:
Ok, fixed. The code is checked in. Thanks